backend/homework/1/app/src/main/java/org/JFPart1/Main.java

```diff
@@ -0,0 +1,119 @@
+package org.JFPart1;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        Scanner sc = new Scanner(System.in);
+
+        LoggingClass loggingClass = new LoggingClass();
+        StudentRepository studentRepository = new StudentRepository();
+
+        System.out.println("Hello! Its a simple application to manage student information");
+
+        boolean continued = true;
+        while (continued){
+
+            System.out.println("Enter 1,2,3 or 4 based on below options:");
+            System.out.println("1. Add new Student to repository");
+            System.out.println("2. Update a Student information");
+            System.out.println("3. Retrieve any Student information");
+            System.out.println("4. Retrieve all Student information");
+            System.out.println("5. Close program");
+
+            int option = checkNumber(sc.nextLine());
+            Student student;
+            boolean isSuccessful = false;
+
+            switch(option){
+                case 1:
+                    student = inputStudent();
+                    isSuccessful = studentRepository.addStudent(student);
+                    if(isSuccessful)loggingClass.addedSuccessfully(student);
+                    else loggingClass.repeatedStudent(student);
+                    break;
+
+                case 2:
+                    student = inputStudent();
+                    isSuccessful = studentRepository.updateStudent(student);
+                    if (isSuccessful)loggingClass.updatedStudent(student);
+                    else loggingClass.studentNotFound(student.getId());
+                    break;
+
+                case 3:
+                    System.out.println("\n\nEnter how do you wanna retrieve student,");
+                    System.out.println("1 for using Student id");
+                    System.out.println("2 for using Student name");
+                    int input = checkNumber(sc.nextLine());
+                    if(input == 1){
+                        System.out.print("Enter id: ");
+                        int id = checkNumber(sc.nextLine());
+                        Student s = studentRepository.getStudent(id);
+                        if(s == null)loggingClass.studentNotFound(id);
```

Comment:
Pls do not tightly couple logger class with different statements but keep it loosely coupled. just export logger and put your error messages.